Tests/DiligentCoreAPITest/src/ClearRenderTargetTest.cpp

```diff
@@ -121,7 +121,7 @@ TEST(ClearRenderTargetTest, AsRenderTarget)
     auto* pEnv       = TestingEnvironment::GetInstance();
     auto* pDevice    = pEnv->GetDevice();
     auto* pSwapChain = pEnv->GetSwapChain();
-    auto* pContext   = pEnv->GetDeviceContext();
```

Comment:
Here and below pContext is the same, so the second definition should simply be removed

Reply:
:grimacing: Sorry made a mess of this should have seen that.